README.md

```diff
@@ -437,6 +437,9 @@ Run `make test` to invoke the test suite, found in the test-suite subdirectory.
 - Iulia Tamas
 - Andrew Twyman
 
+## Contributors
```

Comment:
Is this coming from this policy: https://opensource.google.com/docs/patching/#how-to-patch

If so, that seems to suggest "If copyright authors are listed in a LICENSE, COPYING, AUTHORS", which wouldn't apply here.  We could work on changing that if you'd like, but may not need to delay this diff on it.

It may be that the current list of "Authors" is poorly labeled/named.  It's a list of "primary" contributors, not all contributors.  We've used it as a way to recognize people who wrote large chunks of the tool, or whole new features.  It's not intended to be a claim of copyright, since the whole project is copyrighted by Dropbox, per the CLA.